cmd/koord-manager/extensions/extension.go

```diff
@@ -19,8 +19,24 @@ package extensions
 import (
 	"context"
 
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/rest"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/koordinator-sh/koordinator/pkg/slo-controller/nodemetric"
+	"github.com/koordinator-sh/koordinator/pkg/slo-controller/noderesource"
+	"github.com/koordinator-sh/koordinator/pkg/slo-controller/nodeslo"
+)
+
+var (
```

Comment:
options.go
extensions pkg will be deprecated in future

Reply:
done